modules/claims_api/app/services/claims_api/disability_compensation/pdf_generation_service.rb

```diff
@@ -46,7 +46,7 @@ def generate(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
 
         log_job_progress(auto_claim.id, '526EZ PDF generator job finished', auto_claim.transaction_id)
 
-        auto_claim.id
+        auto_claim.status
```

Comment:
we weren't using the claim id anywhere, so changing to status. could be worth refactoring later to return nil and create attr_reader methods for attributes we want.